oshdb-contributions/src/main/java/InteractionsBuilder.java

```diff
@@ -94,8 +96,9 @@ public static void main(String[] args) {
             String configJSONString = new String( Files.readAllBytes(Paths.get(configFilePath)) );
             JSONObject config = JSON.parseObject(configJSONString);
 
-            h2Path = Paths.get(config.get("oshdb").toString());
-            System.err.println("Will use oshdb: " + h2Path.toString());
+            h2GridPath = config.get("oshdb").toString();
+            h2KeytablesPath = config.get("keytables").toString();
```

Comment:
you need to update your config as well, to include a separate keytable path, (could be the same for "all in one" oshdbs)